data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -445,7 +445,7 @@ Any units adjacent to this unit will fight as if it were dusk when it is night,
         name = _"nova"
         description= _ "This weapon, when used offensively, deals damage to all units adjacent to the caster when it hits." + "
 
-" + _ "This attack is the third verse of the Song of Sun Ascension. After using it, the first verse is availaibe in the next turn. Furthermore, invoking this attack will grant illumination for three turns."
+" + _ "This attack is the third verse of the Song of Sun Ascension. After using it, the first verse is available in the next turn. Furthermore, invoking this attack will grant illumination for three turns."
```

Comment:
Didn't you address this in the other pr?

Reply:
See remark in the original post - the PR is based on the branch for #7964 to avoid merge conflicts.